src/config/backpack/operations/form.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * Default configurations for custom form operations.
+ */
+
+return [
+    // Define the size/looks of the content div for all CRUDs
+    // To override per view use $this->crud->setCreateContentClass('class-string')
+    'contentClass' => 'col-md-12 bold-labels',
+
+    // When using tabbed forms (create & update), what kind of tabs would you like?
+    'tabsType' => 'horizontal', //options: horizontal, vertical
+
+    // How would you like the validation errors to be shown?
+    'groupedErrors' => true,
+    'inlineErrors'  => true,
+
+    // when the page loads, put the cursor on the first input?
+    'autoFocusOnFirstField' => true,
+
+    // Where do you want to redirect the user by default, save?
+    // options: save_and_back, save_and_edit, save_and_new
+    'defaultSaveAction' => 'save_and_back',
+
+    // When the user chooses "save and back" or "save and new", show a bubble
+    // for the fact that the default save action has been changed?
+    'showSaveActionChange' => true, //options: true, false
```

Comment:
Since there is only one save action, shouldn't we make this false ?

Reply:
Sure!

```suggestion
    'showSaveActionChange' => false, //options: true, false
```